public/utils/contextMenu/getActions.tsx

```diff
@@ -74,6 +75,18 @@ export const getActions = ({ core, plugins }) => {
       order: 99,
       onClick: getOnClick('associated'),
     },
+    {
+      id: 'documentationAnomalyDetector',
+      title: <DocumentationTitle />,
+      icon: 'documentation' as EuiIconType,
+      order: 98,
+      onExecute: () => {
+        window.open(
+          'https://opensearch.org/docs/latest/monitoring-plugins/anomaly-detection/index/',
```

Comment:
Let's leverage BASE_DOCS_LINK here: https://github.com/opensearch-project/anomaly-detection-dashboards-plugin/blob/main/public/utils/constants.ts#L54

Reply:
And if the docs link needs to be updated, can you help update?

Reply:
Changed to use the doc link constant. No need to update here actually.